agent/config/config_test.go

```diff
@@ -369,6 +371,14 @@ func TestImageCleanupMinimumNumImagesToDeletePerCycle(t *testing.T) {
 	assert.Equal(t, cfg.NumImagesToDeletePerCycle, DefaultNumImagesToDeletePerCycle, "Wrong value for NumImagesToDeletePerCycle")
 }
 
+func TestInvalidAgentPullBehavior(t *testing.T) {
```

Comment:
can you add a test for just `getAgentPullBehavior()` as well? Make it a table driven test so that you can test that all 4 valid values and an invalid value for `ECS_AGENT_PULL_BEHAVIOR` return the expected output.

Reply:
OK